### PR TITLE
Fixed ListIndexes treating file as index candidates

### PR DESCRIPTION
**Pull request by lauchokyip**
Fixes #...https://github.com/kubernetes-sigs/krew/issues/683
Related issue: #... ListIndexes treating file as index candidates

### CODE REVIEW

File: internal/index/indexoperations/index.go [hunk 1]
```diff
@@ -42,8 +42,11 @@ func ListIndexes(paths environment.Paths) ([]Index, error) {
 	}
 
 	indexes := []Index{}
-	for _, dir := range dirs {
-		indexName := dir.Name()
+	for _, file := range dirs {
```
Comment (corneliusweig) [inline internal/index/indexoperations/index.go:45]: It is confusing that `dirs` does not contain directories but also files. Let's make this clear by renaming to `entries`.

```suggestion
	for _, e := range entries {
```

**Review by corneliusweig — COMMENTED** [review]
Just a minor naming improvements, otherwise LGTM

File: internal/index/indexoperations/index.go (hunk 1 above)
Comment (lauchokyip) [reply internal/index/indexoperations/index.go:45]: Thanks, will work on it now :)

**Comment by lauchokyip** [comment]
@corneliusweig Fixed 👍🏼

**Comment by corneliusweig** [comment]
Thanks!
/lgtm
/approve

**Pull request merged**